content/en/docs/_index.md

```diff
@@ -1,20 +1,16 @@
 
 ---
-title: "Tekton Documentation"
+title: "Welcome to Tekton"
 linkTitle: "Documentation"
 weight: 20
 menu:
   main:
     weight: 20
 ---
 
-Tekton is a cloud native continuous integration and delivery (CI/CD)
-solution. It allows developers to build, test, and deploy across cloud
-providers and on-premise systems.
-
-### What's next
-
-To get started, read the contents below.
+Tekton is a cloud-native solution for building CI/CD pipelines. It consists of Tekton Pipelines, which provides
```

Comment:
NIT: this definition of Tekton is not 100% consistent with that on the main page of the website, but I think it looks ok anyways. @vdemeester @bobcatfish @dibyom thoughts?

Reply:
Thank you very much @afrittoli for your kind cooporation!!
I am very grateful that you kindly checked this PR.

Reply:
ping @vdemeester @bobcatfish @dibyom

Reply:
Looks fine to me...couple of minor nits -> maybe call it CI/CD systems instead of CI/CD pipelines, and mention some of the other projects such as Triggers, Results etc.?

Reply:
@dibyom  thank you for the review.  Actually as mentioned  in https://github.com/tektoncd/website/pull/263#issuecomment-811773962　, we try to keep a clear separation with landing page and overview page, so Only a short description is add here.   Uh. I think my comment for this PR make confused, I will change it clearly.

Reply:
@dibyom agree with your comment, I have correct CI/CD pipelines to CI/CD systems.

Reply:
Hi @dibyom thank you very much for your important comment.
This is @kobaji , working with @cclemonzy .

@cclemonzy has corrected CI/CD pipelines to CI/CD systems. If you had any other request in this PR and commented, we would be very happy.

@afrittoli Thank you very much for approving this PR.
We are waiting for the other approvals, right?